client/src/containers/projects/form/setup/restoration-project-detail/index.tsx

```diff
@@ -130,7 +130,7 @@ export default function RestorationProjectDetails() {
           <div className="basis-1/2 space-y-2">
             <FormField
               control={form.control}
-              name="parameters.restorationActivity"
+              name="parameters.tierSelector"
```

Comment:
I am worried about this change, I am not sure if these concepts are equal. Is this change something that stays within the form, or does it get spreaded to the DTO sent to the BE?

Reply:
It's not directly clear in this code snippet, but if you open the file you can see that it was actually a bug. It had the wrong name because the underlying select element is `tierSelector`.

Reply:
But does this change affect the DTO that is sent to the BE?

Reply:
Yes, `parameters.tierSelector` for `Restoration` type

Reply:
I don't understand. We have both properties for restoration projects, restorationActivity AND tierSelector. 
And I also see that both are sent in the DTO.

I think this is an important bit, we should clarify it